desafio8.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
+  (
+    SELECT
+      ContactName
+    FROM
+      w3schools.customers
+    Where
+      customers.CustomerID = w3schools.orders.CustomerID
+  ) AS `Nome de contato`,
+  (
+    SELECT
+      ShipperName
+    FROM
+      w3schools.shippers
+    Where
+      shippers.ShipperID = w3schools.orders.ShipperID
+  ) AS `Empresa que fez o envio`,
+  OrderDate AS `Data do pedido`
+FROM
+  w3schools.orders
+WHERE
+  ShipperID = 1
+  OR ShipperID = 2
```

Comment:
Aqui você poderia usar o IN também ao inves de OR.
```suggestion
WHERE ShipperID IN (1, 2)
```

Reply:
Bem melhor mesmo, agradeço pela indicação